logicrunner/logicrunner_unit_test.go

```diff
@@ -908,16 +908,6 @@ func (s *LRUnsafeGetLedgerPendingRequestTestSuite) AfterTest(suiteName, testName
 	s.LogicRunnerCommonTestSuite.AfterTest(suiteName, testName)
 }
 
-func (s *LRUnsafeGetLedgerPendingRequestTestSuite) TestAlreadyHaveLedgerQueueElement() {
-	es := &ExecutionState{
-		Ref:                s.ref,
-		Behaviour:          &ValidationSaver{},
-		LedgerQueueElement: &ExecutionQueueElement{pulse: s.currentPulseNumber},
-	}
-	s.lr.unsafeGetLedgerPendingRequest(s.ctx, es)
-	s.Require().Equal(es.LedgerQueueElement.pulse, s.currentPulseNumber)
```

Comment:
that was check, that unsafeGetLedgerPendingRequest does not change ExecutionState. LedgerQueueElement. Need to change LedgerQueueElement to something with expected Parcel or Request or both and check that it does not change